compose/final/app/src/main/graphql/BookTrip.graphql

```diff
@@ -4,6 +4,7 @@ mutation BookTrip($id: ID!) {
         message
         launches {
             id
+            isBooked
```

Comment:
https://www.apollographql.com/docs/kotlin/caching/query-watchers/#updating-the-cache-after-a-mutation

> To update the cache automatically with the new value, you need to request all modified fields in the response: